app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -63,6 +65,12 @@ def in_person_enrollment?(user)
         ProofingComponent.find_by(user: user)&.document_check == Idp::Constants::Vendors::USPS
       end
 
+      def device_profiling_failed?(user)
+        return false unless IdentityConfig.store.proofing_device_profiling_decisioning_enabled
+        proofing_component = ProofingComponent.find_by(user: user)
+        proofing_component.threatmetrix_review_status != 'pass'
```

Comment:
ditto about checking if this is nil?

Reply:
Do you mean checking if proofing_component is nil or if threatmetrix_review_status is nil?

Reply:
I meant `threatmetrix_review_status` is nil

Reply:
I have a test for review status being nil and it returns `true` . I feel like an extra line checking for nil is unnecessary unless we change `threatmetrix_review_status` from 'pass' or 'fail' to a boolean in the future.

Reply:
ah I see, so my expectation is that review status being nil is *not* a failure, that this method should return falsy in that case? Becaue we don't know that it's a for sure negative? and there may be users who fit into a gap of "the feature flag was enabled but we didn't run the check for them" so they don't have data

Reply:
ah so if it is nil let them through?

Reply:
yeah that's my expectation of how we could ensure the safest/smoothest launch

Reply:
I'd like to add my support to the idea that a missing `review_status` in the API response (which I assume is what a `nil` value for `proofing_component..threatmetrix_review_status` indicates) should have (for now) the same behavior as `pass`.